front/src/components/ArticleCreate.jsx

```diff
@@ -37,7 +37,7 @@ export default function ArticleCreate ({ onSubmit }) {
         setTags(tags)
       } catch (err) {
         setToast({
-          text: `Unable to load tags: ${err}`,
+          text: t('articleCreate.notification.errorTags')`: ${err}`,
```

Comment:
I think we should not include the component name:
```suggestion
          text: t('article.getTags.error')`: ${err}`,
```

Reply:
il faut passer le `err` (message d'erreur) en tant qu'argument